internal/layer/sbom_restorer.go

```diff
@@ -41,14 +41,14 @@ func NewSBOMRestorer(opts SBOMRestorerOpts, platformAPI *api.Version) SBOMRestor
 		return &NopSBOMRestorer{}
 	}
 	return &DefaultSBOMRestorer{
-		layersDir: opts.LayersDir,
-		logger:    opts.Logger,
+		LayersDir: opts.LayersDir,
+		Logger:    opts.Logger,
```

Comment:
These could maybe be made private

Reply:
Ah - I made them public so they could be tested here: https://github.com/buildpacks/lifecycle/blob/8b9e787a0028978da28976fc6d63a4de82bc9dc0/analyzer_test.go#L104-L105